specification/tags.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "name": "ServiceRates",
+    "description": ""
+  },
```

Comment:
Let's move this down a little, so it's between `PickupDropoffLocations` and `Shipments`